go.sum

```diff
@@ -744,8 +744,9 @@ github.com/prometheus/client_golang v0.9.3/go.mod h1:/TN21ttK/J9q6uSwhBd54HahCDf
 github.com/prometheus/client_golang v1.0.0/go.mod h1:db9x61etRT2tGnBNRi70OPL5FsnadC4Ky3P0J6CfImo=
 github.com/prometheus/client_golang v1.1.0/go.mod h1:I1FGZT9+L76gKKOs5djB6ezCbFQP1xR9D75/vuwEF3g=
 github.com/prometheus/client_golang v1.7.1/go.mod h1:PY5Wy2awLA44sXw4AOSfFBetzPP4j5+D6mVACh+pe2M=
```

Comment:
Seems like we have to use `replace`

Reply:
right!
[looks](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/3d69f4aad74da5c05628ca2f782830cc95e18db0/go.sum#L738) better now. [Done](https://github.com/k8snetworkplumbingwg/kubemacpool/compare/e0f54533f2da93f89ae48dc77cd9236849e60396..3d69f4aad74da5c05628ca2f782830cc95e18db0)

Reply:
wonder if the combination of require after `go mod tidy --compact` would fix it as well ?
I think it did fix for me one time

Reply:
is <VERSION> the golang version? how is it related? this bump is of an external library. could you explain?

Reply:
yes, golang version
see here please https://github.com/nmstate/kubernetes-nmstate/blob/main/Makefile#L197

It might be not related, but i think it might worth to check it, for example if each go version had the specific version hash that it included, and once you tidy per specific version it cleans them all
I might saw it happen locally one time, so maybe worth to check, as it is pretty easy to do please

Reply:
I think it's only relevant to standard golang libraries, but can't hurt to check